crates/turborepo-lib/src/tracing.rs

```diff
@@ -189,10 +189,14 @@ impl TurboSubscriber {
 
     /// Enables chrome tracing.
     #[tracing::instrument(skip(self, to_file))]
-    pub fn enable_chrome_tracing<P: AsRef<Path>>(&self, to_file: P) -> Result<(), Error> {
+    pub fn enable_chrome_tracing<P: AsRef<Path>>(
+        &self,
+        to_file: P,
+        include_args: bool,
+    ) -> Result<(), Error> {
         let (layer, guard) = tracing_chrome::ChromeLayerBuilder::new()
             .file(to_file)
-            .include_args(true)
+            .include_args(include_args)
```

Comment:
Haven't looked at the chrome traces in a while, but will this still include any package or task names?

Reply:
It will only include the function names. We will not be able to get access to and function arguments so it is likely that packages and tasks will not appear. It _will_ however let us see what parts of the run are taking time.

I mentioned in a thread it may be possible to use valuable with some type-aware filter and some newtype so we can filter out sensitive data

```rust
tracing::log!("running task {} with absolute path {}", task_name, Sensitive(absolute_path));
```